boards/BMS/Inc/Io/Io_Temperatures.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "App_SharedExitCode.h"
+
+/**
+ * Read internal die temperatures for all cell monitoring chips.
+ * @return EXIT_CODE_OK if internal die temperatures were acquired successfully
+ * from all cell monitoring chips. Else, EXIT_CODE_ERROR.
+ */
+ExitCode Io_Temperatures_ReadDieTemperatures(void);
+
+/**
+ * Get the pointer to the array containing internal die temperatures read
+ * from the cell monitoring chips.
+ * @note The subscript value of the array is the number of cell monitoring
```

Comment:
I outlined the array size in this comment @garethellis0 .

Reply:
"The length of the array is equal to the number of cell monitoring devices on the daisy chain" 
or
"The length of the array is equal to the number of cell monitoring devices on the daisy chain ( NUM_OF_CELL_MONITORING_CHIPS)" ?
or 
"The array length is NUM_OF_CELL_MONITORING_CHIPS"
or
"The array length is equal to the number of cell monitoring devices on the daisy chain" 
Possibly? Or something along those terms